packages/api-contracts/src/types/page-scan.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ReportData, ScanStatus } from 'storage-documents';
+import { Page } from './page';
+
+export interface PageScan {
+    id: string;
+    websiteScanId?: string;
+    page: Page;
+    priority: number;
+    scanStatus: ScanStatus;
+    completedTimestamp?: number;
+    resultsBlobId?: string;
```

Comment:
Can be array of ids if there is a complex scan. Since the report falls into same category (blob item) we can join result and report into type below. We can do this change in other PR

```
interface blobResult {
resultKind: string
resultBlobId: string
href?: string
}
```

Reply:
Will make this change in a separate PR